composer.json

```diff
@@ -25,7 +25,7 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 ||^8.0",
```

Comment:
Please use `~8.0.0` as described in the initial issue.
We decided in an internal discussion to use this constraint to avoid BC breaks with newer PHP versions.